Workbooks/Images/Logos/ThycoticLogo.svg

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
The logo file is not valid according to the creation guide. Please check the previous comment.